scripts/docs/Dockerfile

```diff
@@ -25,7 +25,7 @@ RUN apt-get -qqy update && \
 
 RUN ln -fs /usr/bin/python3.7 /usr/bin/python
 
-RUN curl -Lo /usr/local/bin/bazel https://github.com/bazelbuild/bazelisk/releases/download/v1.3.0/bazelisk-linux-amd64 && \
+RUN curl -f -Lo /usr/local/bin/bazel https://github.com/bazelbuild/bazelisk/releases/download/v1.3.0/bazelisk-linux-amd64 && \
```

Comment:
in general you can combine these short binary options e.g. `curl -fLo /usr/local/bin/bazel ...`

Reply:
Done.